doc/development.md

```diff
@@ -39,6 +34,8 @@ docker-compose up -d
 
 ### Initialize the database
 
+NOTE: In dev this will deploy the Sqlite DB, in all other environments we deploy the db using the [usaon-vta-db project](https://github.com/nsidc/usaon-vta-db).
```

Comment:
```suggestion
NOTE: In dev this will initialize the SQLite DB. In all other environments we deploy the db using the [usaon-vta-db project](https://github.com/nsidc/usaon-vta-db).
```